dolly-backend-app/src/main/java/no/nav/dolly/provider/api/OrganisasjonController.java

```diff
@@ -86,4 +90,12 @@ public RsOrganisasjonBestillingStatus hentBestilling(
 
         return bestillingService.fetchBestillingStatusByBrukerId(brukerId);
     }
+
+    @CacheEvict(value = CACHE_ORG_BESTILLING, allEntries = true)
```

Comment:
Alternativt kunne du slettet per orgnummer som ville ha motsvart sletting for testpersoner.
Og samtidig slette alle tilhørende bestillinger, inkludert gjenopprettinger.

Reply:
Ja veldig greit å ha det likt, har lagt til muligheten for det nå 😄